build.gradle

```diff
@@ -564,6 +568,12 @@ project('shared:controller-api') {
 }
 
 project('controller') {
+    
+    //TODO: Work around deprication warning in generated code caused by https://github.com/FasterXML/jackson-databind/issues/1786
```

Comment:
`deprecation`

Reply:
The content referenced by the link does not seem to explain why this is needed. I guess we don't want the build to fail on warnings that indicate deprecated code, here. Rewording the comment should help.

Reply:
I have changed the code to suppress only the one file.